spec/support/resource_testing.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.shared_context 'resource testing' do |parameter|
```

Comment:
This will eventually get moved out and serve as the basis of Resource integration specs.